tasks/project.yml

```diff
@@ -34,13 +34,21 @@
   tags:
     - update_react_repo
 
+- name: Install g++ package
+  apt:
+    name: "g++"
+    update_cache: yes
+  become: yes
+  tags:
+    - install_compiler
+
 - name: Install node dependencies
-  command: "yarn --production"
+  command: "yarn install"
   args:
     chdir: "{{ repo_path }}/{{ github_repo | basename }}"
   become: yes
   become_user: "{{ server_user }}"
-  when: react_production
+  # when: react_production
```

Comment:
please remove line. this taks should be executed no matter what environment we are working on